components/eam/src/chemistry/modal_aero/modal_aero_drydep.F90

```diff
@@ -733,4 +615,380 @@ subroutine calcram(ncol,landfrac,icefrac,ocnfrac, &
+             ! Total resistence and deposition velocity of turbulent deposition,
+             ! see Eq. (21) of Zender et al. (2003)
+             !--------------------------------------------------------------------
+             rss_trb = ram1(ii) + rss_lmn + ram1(ii)*rss_lmn*vlc_grv(ii)
+             vlc_trb_ontype = 1.0_r8 / rss_trb
+
+             !--------------------------------------------------------------------------------
+             ! Contributions to the single-value bulk deposition velocities of the grid cell
+             !--------------------------------------------------------------------------------
+             vlc_trb_wgtsum = vlc_trb_wgtsum + lnd_frc*( vlc_trb_ontype )
+             vlc_dry_wgtsum = vlc_dry_wgtsum + lnd_frc*( vlc_trb_ontype + vlc_grv(ii) )
+          endif
+
+       enddo  ! lt=1,n_land_type
+
+       vlc_trb(ii) = vlc_trb_wgtsum
+       vlc_dry(ii) = vlc_dry_wgtsum
+
+    enddo ! ii=1,ncol
+    !-------
+
+  end subroutine modal_aero_turb_drydep_velocity
+
+  !==============================================================================
+  ! Calculate the radius for a moment of a lognormal size distribution
+  !==============================================================================
+  real(r8) function radius_for_moment( moment,sig_part,radius_part,radius_max)
+
+    integer, intent(in) :: moment        ! moment of the particle size distribution. 0 = number; 3 = volume
+    real(r8),intent(in) :: sig_part      ! geometric standard deviation of particle size distribution
+    real(r8),intent(in) :: radius_part   ! mean (volume or number) particle radius [m]
+    real(r8),intent(in) :: radius_max    ! developer-specified upper bound of mean radius [m]
+
+    real(r8) :: lnsig
+    
+    lnsig = log(sig_part)
+    radius_for_moment = min(radius_max,radius_part)*exp((float(moment)-1.5_r8)*lnsig*lnsig)
+
+  end function radius_for_moment
+
+  !==========================================================================
+  ! Calculate dynamic viscosity of air, unit [kg m-1 s-1]. See RoY94 p. 102
+  !==========================================================================
+  real(r8) function air_dynamic_viscosity( temp )
+
+    real(r8),intent(in) :: temp   ! air temperature [K]
+
+    air_dynamic_viscosity = 1.72e-5_r8 * ( (temp/273.0_r8)**1.5_r8) * 393.0_r8 &
+                                          /(temp+120.0_r8)
+
+  end function air_dynamic_viscosity
+
+  !==========================================================================
+  ! Calculate kinematic viscosity of air, unit [m2 s-1]
+  !==========================================================================
+  real(r8) function air_kinematic_viscosity( temp, pres, rair )
+
+    real(r8),intent(in) :: temp   ! air temperature [K]
+    real(r8),intent(in) :: pres   ! air pressure [Pa]
+    real(r8),intent(in) :: rair   ! gas constant of air [J/K/kg]
+
+    real(r8) :: vsc_dyn_atm  ! kinematic viscosity of air [m2 s-1]
```

Comment:
the comment of variable name and unit is incorrect

Reply:
Thanks! Corrected in [6a6bd6c](https://github.com/eagles-project/e3sm_mam4_refactor/pull/68/commits/6a6bd6c2d29e8411589ca4f23ef70f5d8c709ede)